airflow-core/src/airflow/api_fastapi/core_api/datamodels/assets.py

```diff
@@ -160,6 +160,20 @@ class QueuedEventCollectionResponse(BaseModel):
     total_entries: int
 
 
+class CreateAssetBody(StrictBaseModel):
+    """Create asset request."""
+
+    name: str = Field(..., min_length=1, max_length=1500)
+    uri: str = Field(..., min_length=1, max_length=1500)
+    group: str = Field(default="", max_length=1500)
```

Comment:
You don't need to do that. 

The DB will refuse such wrong input (if strings are too long),  then the AIP & UI should gracefully handle and display this error.

Can you update and confirm please?

```suggestion
    name: str
    uri: str
    group: str
```